src/index.html

```diff
@@ -6,13 +6,70 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="product">
+        <img
+          src="images/imac.jpeg"
+          alt="logo"
+          class="product__image"
+        />
+
+        <div class="product__name">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product__code">Product code: 195434</div>
+
+        <div class="rating">
```

Comment:
make it an element of product BEM block
```suggestion
        <div class="product__rating">
```